src/main/webapp/js/instructorStudentRecords.es6

```diff
@@ -122,6 +126,8 @@ function checkComment(form) {
     }
 }
 
+/* eslint-enable consistent-return */
```

Comment:
I know about the `e.stopPropagation()` thing, but I'm pretty sure most of the time whoever put `return false;` there only intended it to be `e.preventDefault()`.

Of course exceptions happen. Do check out if `e.stopPropagation()` is also needed.